packages/e2e-tests/specs/splitting-merging.test.js

```diff
@@ -210,4 +210,24 @@ describe( 'splitting and merging blocks', () => {
 
 		expect( isInDefaultBlock ).toBe( true );
 	} );
+
+	it( 'should create a paragraph block above when pressing enter at start of heading', async () => {
+		await insertBlock( 'Heading' );
+		await page.keyboard.type( 'Abc' );
+		await pressKeyTimes( 'ArrowLeft', 3 );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.press( 'ArrowUp' );
+
+		const activeElementClassList = await page.evaluate( () => document.activeElement.classList );
+		expect( Object.values( activeElementClassList ) ).toContain( 'wp-block-paragraph' );
```

Comment:
What is the arrow up and active class check testing for? Is checking the content not enough?

Reply:
I was initially concerned with checking the active element of the caret position, but I think you're right, if we're checking the content, the active class check is overkill, I'll remove it.